articles/service-fabric-mesh/service-fabric-mesh-howto-deploy-app-reliabledisk-volume.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Store state in an Azure Service Fabric Mesh application by mounting Service Fabric Reliable Disk based volume inside the container | Microsoft Docs
+description: Learn how to store state in an Azure Service Fabric Mesh application by mounting Service Fabric Reliable Disk based volume inside the container using the Azure CLI.
+services: service-fabric-mesh
+documentationcenter: .net
+author: ashishnegi
+manager: raunakpandya
+editor: ''
+ms.assetid:
+ms.service: service-fabric-mesh
+ms.devlang: azure-cli
+ms.topic: conceptual
+ms.tgt_pltfrm: NA
+ms.workload: NA
+ms.date: 08/09/2018
+ms.author: asnegi
+ms.custom: mvc, devcenter
+---
+
+# Store state in an Azure Service Fabric Mesh application by mounting a highly available Service Fabric Reliable Disk based volume inside the container
+
+The common method of persisting state with container apps is to use remote storage like Azure File Storage or database like Azure Cosmos DB. This incurs significant read and write network latency to the remote store.
+
+This article shows how to store state in highly available Service Fabric Reliable Disk by mounting a volume inside the container of a Service Fabric Mesh application.
+Service Fabric Reliable Disk provides volumes for local reads with writes replicated within the Service Fabric Cluster for high availability. This removes network calls for reads and reduces network latency for writes. If the container restarts or moves to another node, new container instance will see the same volume as older one. Thus it is both efficient and highly available.
```

Comment:
"This removes latency in reads and writes."

Reply:
"...same volume as the on the original node, making the volume highly available."